packages/react-components/react-combobox/src/components/Option/useOptionStyles.ts

```diff
@@ -86,18 +86,30 @@ const useStyles = makeStyles({
     },
   },
 
-  multiselectCheck: {
-    color: tokens.colorNeutralForeground3,
-    fontSize: tokens.fontSizeBase500,
+  selectedCheck: {
     visibility: 'visible',
   },
 
-  selectedCheck: {
+  multiselectCheck: {
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralForeground3),
+    ...shorthands.borderRadius(tokens.borderRadiusSmall),
+    boxSizing: 'border-box',
+
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+
+    fill: 'currentColor',
+    fontSize: '12px',
+    height: '16px',
+    width: '16px',
     visibility: 'visible',
   },
 
   selectedMultiselectCheck: {
-    color: tokens.colorBrandBackground,
+    backgroundColor: tokens.colorCompoundBrandBackground,
+    color: tokens.colorNeutralForegroundInverted,
+    ...shorthands.borderColor(tokens.colorCompoundBrandBackground),
```

Comment:
Don't you need to add `:hover` and `:active` styles as well?

Reply:
It's a bit different, since the Option itself has a hover and active style, rather than the checkbox indicator

Reply:
I double checked with Yvonne, and she confirmed the hover/active are different from Checkbox because of the changing backplate :)